lookout/style/format/tests/code_generator_data.py

```diff
@@ -1,8 +1,7 @@
 from collections import OrderedDict
 
-from lookout.style.format.feature_utils import (
-    CLASS_INDEX, CLS_DOUBLE_QUOTE, CLS_NEWLINE, CLS_NOOP, CLS_SINGLE_QUOTE, CLS_SPACE,
-    CLS_SPACE_DEC, CLS_SPACE_INC)
+from lookout.style.format.classes import CLS_SPACE, CLS_NEWLINE, CLS_SPACE_INC, CLS_SPACE_DEC, \
+    CLS_SINGLE_QUOTE, CLS_DOUBLE_QUOTE, CLS_NOOP, CLASS_INDEX
```

Comment:
why? I remember we decided to use `()` everywhere.

Reply:
We need to write this to our style guide explicitly... I don't care TBH, so I can switch back to braces.

Reply:
Ah now I see: this is how PyCharm autoformats imports. So maybe we should allow both styles then

Reply:
now I think to write a plugin to fix imports to use braces only and do not have this issue with CamelCase sorting.